static/audiofeed.yaml

```
@@ -1,3 +1,19 @@
+- title: 'Fantastische Fakten 2'
+  date: 2025-08-03T19:00
+  image: https://raw.githubusercontent.com/LucaNerlich/m10z/main/static/img/formate/cover/fakten.jpg
+  description: |2-
+                  Sind alle stillen Hügel gruselig? Sind Videospielcharaktere nach euch benannt? Wie war das noch mit der langsamen Musik? Ihr wisst es nicht? In unserem Format „Fantastische Fakten“ könnt ihr es herausfinden.
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)